examples/train_model.py

```diff
@@ -62,6 +62,9 @@ def setup_args(model_args=None):
                        type=float, default=1.0,
                        help='value at which training will stop if exceeded by '
                             'training metric')
+    train.add_argument('--save-last-valid', type='bool', default=False
+                       help='Saves the model to model_file.last_valid after '
+                            'every validation (overwriting previous saves).')
```

Comment:
needs to explain the best model is still saved in model_file

Reply:
in fact i think something  like  'save-checkpoint'  might be clearer?  then the save-every-n-secs should save there too (and call it   .checkpt or something instead?)